stable/ambassador/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
-appVersion: 0.50.2
+appVersion: 0.50.3
```

Comment:
Need to bump version in values.yaml and in README

Reply:
Oh dear. Thanks for the catch!!